src/Microsoft.Azure.WebJobs.Host/Scale/ScaleLoggerExtension.cs

```diff
@@ -82,11 +89,19 @@ public static void LogFunctionScaleVote(this ILogger logger, string functionName
         /// <summary>
         /// Logs an error that occurred while scaling a function, including the function name and exception details.
         /// </summary>
-        internal static void LogFunctionScaleError(this ILogger logger, string message, string functionName, Exception ex)
+        public static void LogFunctionScaleError(this ILogger logger, string message, string functionName, Exception ex)
         {
             _logFunctionScaleError(logger, functionName, message, ex);
         }
 
+        /// <summary>
+        /// Logs an warning that occurred while scaling a function, including the function name and exception details.
+        /// </summary>
+        public static void LogFunctionScaleWarning(this ILogger logger, string message, string functionName, Exception ex)
```

Comment:
Thinking exception should be an optional param? I see in your CosmosDB PR you're having to pass an explicit null value